x-pack/solutions/security/plugins/security_solution/server/lib/telemetry/tasks/filterlists.ts

```diff
@@ -48,14 +48,14 @@ export function createTelemetryFilterListArtifactTaskConfig() {
         }
 
         const artifact = manifest.data as unknown as TelemetryFilterListArtifact;
-        log.l('New filterlist artifact', { artifact });
+        log.debug('New filterlist artifact', { artifact } as LogMeta);
         filterList.endpointAlerts = artifact.endpoint_alerts;
         filterList.exceptionLists = artifact.exception_lists;
         filterList.prebuiltRulesAlerts = artifact.prebuilt_rules_alerts;
         await taskMetricsService.end(trace);
         return 0;
       } catch (err) {
-        log.l('Failed to set telemetry filterlist artifact', { error: err.message });
+        log.warn('Failed to set telemetry filterlist artifact', { error: err.message } as LogMeta);
```

Comment:
To be ECS compliant, it should be:
```suggestion
        log.warn('Failed to set telemetry filterlist artifact', { error: { message: err.message } });
```